src/explore-education-statistics-admin/src/pages/bau/BauImportsPage.tsx

```diff
@@ -0,0 +1,94 @@
+import Link from '@admin/components/Link';
+import Page from '@admin/components/Page';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useAsyncRetry from '@common/hooks/useAsyncRetry';
+import React from 'react';
+import importStatusBauService from '@admin/services/importStatusBauService';
+
+const BauImportsPage = () => {
+  const { value, isLoading } = useAsyncRetry(() =>
+    importStatusBauService.getAllIncompleteImports(),
+  );
+
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        { name: 'Platform administration', link: '/administration' },
+        { name: 'Incomplete imports' },
+      ]}
+    >
+      <h1 className="govuk-heading-xl">
+        <span className="govuk-caption-xl">Manage incomplete imports</span>
+        Incomplete imports
+      </h1>
+      <table className="govuk-table">
```

Comment:
No need for any of the `govuk-table` classes for the entirety of this table. Tables already have these styles applied, so they're just redundant and increase code size.